src/register/register-gnome/gnucash-item-edit.c

```diff
@@ -505,6 +505,9 @@ gnc_item_edit_new (GnucashSheet *sheet)
                            NULL);
     gtk_layout_put (GTK_LAYOUT(sheet), GTK_WIDGET(item_edit), 0, 0);
 
+    // Set the style context for this widget so it can be easily manipulated with css
+    gtk_style_context_add_class (gtk_widget_get_style_context (GTK_WIDGET(item_edit)), "GncRegisterItemEdit");
+
```

Comment:
GncRegisterItemEdit is another name example of describing the implementation instead of the function. This widget is an implementation of a cell cursor (the "purpose"), so use that as css node/class description. The idea is the theme designer is not interested in how something is implemented, but what the function is of a given node so the defined style can support this function. And ideally it should not matter how we implement it internally and the chosen style should work equally well regardless of whether the register is implemented using gtklayout/gtkentry or gtktreeview. (That's unlikely to realize fully, but it helps to think in those terms to better separate function from implementation).